specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/examples/CreateACustomImageScaleSetFromAnUnmanagedGeneralizedOsImage.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Under 2018-09-30 is there only disk.json.  Remove all example files that are not referred by disk.json.